src/url-handling.js

```diff
@@ -136,7 +196,7 @@ export function urlStateToUrlObject(urlState: UrlState): UrlObject {
   return { query, pathParts };
 }
 
-export function urlFromState(urlState: UrlState): string {
+export function urlStateToLocationString(urlState: UrlState): string {
```

Comment:
I tried to be more consistent and verbose in the naming of the functions, so that everything follows the form `typeAToTypeB`